pkg/cluster/backupstorage/pv.go

```diff
@@ -11,23 +11,25 @@ type pv struct {
 	clusterName   string
 	namespace     string
 	pvProvisioner string
+	storageClass  string
```

Comment:
you can remove the pvProvisioner string actually. i think we actually convert the provisioner to storageclass anyway